cordova-plugin-geolocation.opam

```diff
@@ -2,10 +2,12 @@ opam-version: "2.0"
 name:         "ocaml-cordova-plugin-geolocation"
 maintainer:   "Danny Willems <contact@danny-willems.be>"
 authors:       "Danny Willems <contact@danny-willems.be>"
-homepage:     "https://github.com/dannywillems/ocaml-cordova-plugin-geolocation"
-bug-reports:  "https://github.com/dannywillems/ocaml-cordova-plugin-geolocation/issues"
-dev-repo:     "https://github.com/dannywillems/ocaml-cordova-plugin-geolocation"
+homepage:     "git+https://dannywillems/ocaml-cordova-plugin-geolocation"
+bug-reports:  "git+https://dannywillems/ocaml-cordova-plugin-geolocation/issues"
+dev-repo:     "git+https://dannywillems/ocaml-cordova-plugin-geolocation"
```

Comment:
To be fixed, see https://github.com/besport/ocaml-cordova-plugin-inappbrowser/pull/2#discussion_r681781378.